packages/cardano-services/src/Program/programs/types.ts

```diff
@@ -56,7 +53,8 @@ export enum ProviderServerOptionDescriptions {
   UseKoraLabsProvider = 'Use the KoraLabs handle provider',
   UseQueue = 'Enables RabbitMQ',
   PaginationPageSizeLimit = 'Pagination page size limit shared across all providers',
-  HandleProviderServerUrl = 'URL for the Handle provider server'
+  HandleProviderServerUrl = 'URL for the Handle provider server',
+  useTypeormNftMetadata = 'Use the TypeORM NFT Metadata Service'
```

Comment:
```suggestion
  UseTypeormAssetProvider = 'Use the TypeORM Stake Pool provider (default is db-sync)'
```

Reply:
Fixed on 535e7d9

Reply:
Please update to start with uppercase U so that it's consistent with other key names